src/service/harvests/HarvestsService.ts

```diff
@@ -0,0 +1,23 @@
+import { Service } from '@tsed/common';
+import { GraphQLClient } from 'graphql-request';
+import {
+	getSdk,
+	HarvestsQuery,
+	HarvestsQueryVariables,
+	Sdk as BadgerDaoGraphqlSdk,
+} from '../../graphql/generated/badger-dao';
+import { BADGER_DAO_SUBGRAPH_URL } from '../../util/constants';
+
+@Service()
+export class HarvestsService {
+	private badgerDaoGraphqlSdk: BadgerDaoGraphqlSdk;
+
+	constructor() {
+		const badgerDaoGraphqlClient = new GraphQLClient(BADGER_DAO_SUBGRAPH_URL);
```

Comment:
I think we should probably delete the `GraphService` since we are going this approach but we can skip it here.  just a mental note for myself.

Reply:
can we add a really short readme in this directory just explaing how to query this endpoint while we don't have TSED open API annotations done?